src/components/SwapEtherspotKlimaZap.tsx

```diff
@@ -0,0 +1,1189 @@
+import './SwapEtherspotKlimaZap.css'
```

Comment:
It's hard to find all the Etherspot related code in here. But we gonna test this a lot anyway I guess

Reply:
I hope!